docs/001_develop/03_client-capabilities/010_filters/foundation-filters.mdx

```diff
@@ -0,0 +1,437 @@
+
+```ts
+import { UserTargeting } from '@genesislcap/foundation-filters';
+...
+@UserTargeting userTargeting: UserTargeting;
+...
+const outcome = this.userTargeting.hasAdminProfile();
+```
+
+This is quite a simplistic example, but we hope it helps to highlight that common parameters can be predefined in the
+filters themselves where needed. It is also worth mentioning that DI-based filters are registered as transient, meaning
+you will get a new instance for every dependency request or container.get(). The ClientFilterRunner used to run chained
+filters is also transient.
+
+## Combining filters with the runner method
+
+The `clientFilterRunner` method allows you to run a combination of filters sequentially, evaluating conditions like userAgent, timeWindow, or URL patterns matches. If any filter fails(returns `false`), then the entire result is `false`, enabling managing complex filtering logic in one function.
+<!--TODO:(./docs/api/foundation-filters.clientfilterrunner.md) --> 
+
+### Example
+```ts
+import { ClientFilterRunner } from '@genesislcap/foundation-filters';
+...
+@ClientFilterRunner runner: ClientFilterRunner;
+
+const outcome = this.runner.run([
+  {
+    name: 'userAgent',
+    parameters: {
+      browsers: ['Chrome'],
+    },
+  },
+  {
+    name: 'timeWindow',
+    parameters: {
+      start: '2024-11-21',
+      end: '2024-12-31',
+    },
+  },
+]);
+console.log(outcome); // true if all the filters pass 
+
+```
+
+## Filters explanations with examples:
+
+Let's take a closer look at each filter, understanding their usage with the help of examples.
+
+### nodeEnv
+The nodeEnv filter in Genesis’s foundation filters checks the current application environment (such as "development," "production," etc.) and returns true if the filter’s specified environment matches the current one. This filter ensures that certain features or configurations are available only in intended environments by managing environment-specific conditions.
+
+Lets look at the usage examples as function and when used with DI:
+
+As function:
+```typescript
+import { nodeEnvFilter } from '@genesislcap/foundation-filters';
+...
+const outcome = nodeEnvFilter(['development']);
+```
+As DI:
+```typescript
+import { NodeEnv } from '@genesislcap/foundation-filters';
+...
+@NodeEnv nodeEnv: NodeEnv;
+...
+const outcome = this.nodeEnv.filter({envs:[ 'development']});
+```
+The nodeEnv also provides a `isDevelopment` variable that can be used directly so the above example can be rewritten as:
+```typescript
+import { NodeEnv } from '@genesislcap/foundation-filters';
+...
+@NodeEnv nodeEnv: NodeEnv;
+...
+const outcome = this.nodeEnv.isDevelopment;
+```
+
+you can find other available API for nodeEnv in the API Docs <!-- TODO:enter link here from docs -->
+
+### percentage
```

Comment:
All of these filter titles should either be wrapped in backticks `percentage` or capitalised "Percentage"

Reply:
added back ticks on all